ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -201,6 +203,10 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 		return fmt.Errorf("You must specify 2 comma-separated availability zones with the '--%s' flag", command.VpcAzFlag)
 	}
 
+	if validateCommaSeparatedParam(cfnParams, cloudformation.ParameterKeyInstanceRole, 1, 1) {
+		return fmt.Errorf("You can only specify one instance role ARN with the '--%s' flag", command.InstanceRoleFlag)
```

Comment:
Oops, this should actually just be "instance role name", since we don't accept colons and therefore can't pass in the full role arn. Not sure if we should, but can fix the error message for now.